Java/syntax_test_java.java

```diff
@@ -42,13 +45,14 @@
 //                                                     ^ punctuation.terminator.java
 
     public static void main(String... args) {
-//  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ meta.method
+//                     ^^^^^^^^^^^^^^^^^^^^^^^ meta.method
 //  ^^^^^^ storage.modifier.java
 //         ^^^^^^ storage.modifier.java
-//                ^^^^ meta.method.return-type storage.type
+//                ^^^^ storage.type
 //                     ^^^^ meta.method.identifier.java entity.name.function.java
 //                         ^^^^^^^^^^^^^^^^ meta.method.parameters.java
 //                         ^ punctuation.section.parens.begin.java
+//                                ^^^ storage.modifier.variable-arity
```

Comment:
In other syntaxes this is scoped as `keyword.operator.variadic` (C/C++/Obj-C/Obj-C++) or `keyword.operator` (Rust).

Reply:
Went with keyword.operator.variadic